### PR TITLE
output addition and iPhone 6S+ iOS 11.2.6 addition

### PR DESCRIPTION
**Pull request by Dave1482**
New device and fixed typos

### CODE REVIEW

File: Jailbreak112/jailbreak112/jailbreak.c [hunk 1]
```diff
@@ -53,7 +53,8 @@ int jailbreak()
         printf("realhost: %#llx\n", realhost);
         printf("osdata_get_metaclass: %#llx\n", osdata_get_metaclass);
         printf("osserialize_serialize: %#llx\n", osserializer_serialize);
-        printf("proc_task: %#llx\n\n", proc_task);
+        printf("proc_task: %#llx\n", proc_task);
+        printf("lel0_sync_vec64_long: %#llx\n\n", kernelptr_bazad;
```
Comment (N00byEdge) [inline Jailbreak112/jailbreak112/jailbreak.c]: Missing closing paren?

**Comment by Dave1482** [comment]
Oooo whoops

**Review by Dave1482 — COMMENTED** [review]
Fixed missing parentheses typo

**Comment by Dave1482** [comment]
Thank you N00byEdge

**Comment by MTJailed** [comment]
Thanx

**Pull request merged**